packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -473,6 +475,11 @@ function execFileIgnoreError(
   });
 }
 
+async function getChromiumVersionFromBinary(path: string) {
+  const { stdout } = await execFileIgnoreError(path, ['--versions'], {});
+  return electronToChromium(JSON.parse(stdout).electron);
```

Comment:
I wonder if this can be just this?

```suggestion
  return JSON.parse(stdout).chrome;
```

Reply:
Based on what the package does I think it should be 🤞